Whisper.net/Ggml/WhisperGgmlDownloader.cs

```diff
@@ -4,9 +4,9 @@ namespace Whisper.net.Ggml;
 
 public static class WhisperGgmlDownloader
 {
-    private static readonly Lazy<HttpClient> httpClient = new(() => new HttpClient());
+    private static readonly Lazy<HttpClient> httpClient = new(() => new HttpClient(){Timeout = Timeout.InfiniteTimeSpan});
 
-    public static async Task<Stream> GetGgmlModelAsync(GgmlType type)
+    public static async Task<Stream> GetGgmlModelAsync(GgmlType type, CancellationToken cancellationToken)
```

Comment:
Can you, please make it optional?
e.g. 
`GetGgmlModelAsync(GgmlType type, CancellationToken cancellationToken = default)`

Reply:
Sure! Just done

Reply:
Thank you for your contribution :)